mm2src/common/common.rs

```diff
@@ -117,6 +117,8 @@ pub mod wio;
 
 #[cfg(target_arch = "wasm32")] pub use wasm::*;
 
+#[cfg(target_arch = "wasm32")] use std::path::PathBuf;
```

Comment:
I just had this [error](https://dev.azure.com/ortgma/Marketmaker/_build/results?buildId=8866&view=logs&j=1ca5303f-ca45-5063-fac0-21a06116c5e3&t=9f265683-e290-57d7-97f5-44d3b9a02ad1) and recomendation to import `std::path::PathBuf`. Sorry, what do you mean by condition?

Reply:
`#[cfg(target_arch = "wasm32")]` this is compile-time check which provides conditional compilation.

We have `PathBuf` imported as
```rs
#[cfg(target_arch = "wasm32")] use std::path::PathBuf;
```

and
```rs
cfg_native!{ // checks if target isn't wasm
    use std::path::PathBuf;
}
```

Because we are using it for both targets, we can import it as it is without any compile-time condition.

Reply:
Oh, got it, thanks! Is is imported without condition now.
Done.